hack/deploy.sh

```diff
@@ -0,0 +1,23 @@
+KUBEVIRT="$GOPATH/src/kubevirt.io/kubevirt/"
+KUBECTL="$KUBEVIRT/cluster/kubectl.sh"
+
+# We need to grab absolute path to all manifests since we'll have to pushd
+# later on.
+MANIFESTS=$(find $(pwd)/manifests -name *.yaml)
```

Comment:
This is a total mess, but I understand why we need this temporary solution.

Reply:
@petrkotas make vagrant-deploy can probably be reduced to kubectl apply -f manifests/persistentvm-resource.yaml -- no need for hack-deploy

Reply:
@fabiand that will be doable when followup PR is in as it routes cluster/kubectl.sh to kubevirt's installation